### PR TITLE
Python code for DNA and RNA classes

### PR DESCRIPTION
**Pull request by IrinaBabkina**

### CODE REVIEW

File: Classes_DNA_RNA.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class Rna(str):
+
+    def __init__(self, i):
+        if not all(i in 'ATGCU' for i in self) == True:
```
Comment (mariafiruleva) [inline Classes_DNA_RNA.py]: In this case, Dna object can contain "U" and Rna object can contain "T": try print(type(Dna('AUGC'))) and print(type(Rna('ATGC'))).

File: Classes_DNA_RNA.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            raise ValueError()
```
Comment (mariafiruleva) [inline Classes_DNA_RNA.py]: It is good to give more information for a user: you can add error explanation, e.g., "your sequence contains the inappropriate base"

File: Classes_DNA_RNA.py (hunk 1 above)
Comment (mariafiruleva) [inline Classes_DNA_RNA.py]: It is good to check that user's sequence is not empty because it will provoke errors for class methods: e.g, try Dna('').gc.

Comment (IrinaBabkina) [reply Classes_DNA_RNA.py]: Now only "AUGC" possible for Rna
and "ATCG" for DNA

File: Classes_DNA_RNA.py (hunk 2 above)
Comment (IrinaBabkina) [reply Classes_DNA_RNA.py]: I add this message for exeption. Thanks

File: Classes_DNA_RNA.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+class Rna(str):
```
Comment (mariafiruleva) [inline Classes_DNA_RNA.py]: Good job! May you add a description of your code to README.md file?

File: Classes_DNA_RNA.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            'G': 'C',
+            'C': 'G',
+            'U': 'A',
+            }
+        nucl_rev = []
+
+        for i in list_nucl:
+            nucl_rev.append(nucl_dict[i])
+
+        return ''.join(nucl_rev)
```
[the remark this answers is not in the document]
Comment (IrinaBabkina) [reply Classes_DNA_RNA.py]: I fixed it. Now that`s OK

File: Classes_DNA_RNA.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+class Dna(Rna):
+
+    def __init__(self, i):
+        if self == '':
+            raise ValueError('your sequence is empty')
+        elif not all(i in 'ATGC' for i in self) == True:
+            raise ValueError('your sequence contains the inappropriate base'
+                             )
+
+    def transcribe(self):
```
[the remark this answers is not in the document]
Comment (IrinaBabkina) [reply Classes_DNA_RNA.py]: I add information about input data un README.md file

File: Classes_DNA_RNA.py (hunk 3 above)
Comment (IrinaBabkina) [reply Classes_DNA_RNA.py]: You can read my decrtiption in README.md file now :)

**Pull request closed without merging**